app/services/irs_attempts_api/tracker_events.rb

```diff
@@ -89,6 +89,12 @@ def idv_document_upload_rate_limited
     # @param [String] document_number
     # @param [String] document_issued
     # @param [String] document_expiration
+    # @param [String] front_image
```

Comment:
I think the table of descriptions you have in this PR would make for really good parameter documentation here. 


also especially because it's not clear from just this part of `front_image` is the bytes of the image, or the file name, etc

Reply:
I agree, it would be clearer with, say, `front_image_uuid`